### PR TITLE
Fix /requests and /groupname/manage-subscribers

### PR DESCRIPTION
**Pull request by clbn**

### CODE REVIEW

File: src/components/requests.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {acceptGroupRequest, rejectGroupRequest,
         acceptUserRequest, rejectUserRequest} from '../redux/action-creators'
 import {tileUserListFactory, WITH_REQUEST_HANDLES} from './tile-user-list'
 
-const TileList = tileUserListFactory({type: WITH_REQUEST_HANDLES})
+const TileList = tileUserListFactory({type: WITH_REQUEST_HANDLES, withActions: true})
```
Comment (ujenjt) [inline src/components/requests.jsx]: `withActions` looks redundant in this case, we can git this info directly from `type`. For example: `PLAIN` - without any actions, and `WITH_REQUEST_HANDLES`, `WITH_REMOVE_AND_MAKE_ADMIN_HANDLES`, `WITH_REMOVE_ADMIN_RIGHTS` – it's the types with actions

File: src/components/tile-user-list.jsx [hunk 2]
```diff
@@ -12,7 +13,7 @@ const renderUsers = (type) => (user) => {
         <div className="avatar">
           <img src={user.profilePictureUrl}/>
         </div>
-        <span>{user.screenName}</span>
+        <UserName user={user}/>
```
Comment (ujenjt) [inline src/components/tile-user-list.jsx]: :thumbsup:

File: src/components/requests.jsx (hunk 1 above)
Comment (clbn) [inline src/components/requests.jsx]: Agree. Will update it. Thanks.

**Comment by clbn** [comment]
@ujenjt Updated:
- Addressed you comment (removed `withActions`)
- Added `confirmFirst()` for "Unsub" action

**Pull request merged**